homeassistant/components/renault/quality_scale.yaml

```diff
@@ -47,7 +47,9 @@ rules:
   docs-supported-functions: todo
   docs-troubleshooting: todo
   docs-use-cases: todo
-  dynamic-devices: todo
+  dynamic-devices:
+    status: exempt
+    comment: Adding/removing a vehicle is a rare occurrence
```

Comment:
It doesn't seem useful to trigger a lookup every x minutes for new vehicles - we'd be using up valuable hits and renault is already rate-limited per account.

It's more likely that if someone adds a new car they will just reload the integration

Reply:
See also: https://github.com/home-assistant/core/pull/142964